packages/ui-extensions/src/surfaces/point-of-sale/event/input/TransactionCompleteInput.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  Discount,
+  LineItem,
+  PaymentMethod,
+  ShippingLine,
+  TaxLine,
+} from '../../api';
+import {BaseInput} from './BaseInput';
+
+export interface TransactionCompleteInput extends BaseInput {
+  transactionComplete: {
+    discounts: Discount[];
+    draftCheckoutId: number;
+    lineItems: LineItem[];
+    orderId: number;
+    paymentMethods: PaymentMethod[];
+    shippingLines: ShippingLine;
```

Comment:
I think this is also optional

Reply:
Add optional values for
- orderId - [checkout interface](https://github.com/Shopify/pos-next-react-native/blob/50520de03bd020dcc7fb90bf3cc99b089f9817ad/src/state/Checkout/state.ts#L114)
- shippingLines - [checkout interface](https://github.com/Shopify/pos-next-react-native/blob/50520de03bd020dcc7fb90bf3cc99b089f9817ad/src/state/Checkout/state.ts#L110)
- taxLines - [checkout interface](https://github.com/Shopify/pos-next-react-native/blob/50520de03bd020dcc7fb90bf3cc99b089f9817ad/src/state/Checkout/state.ts#L106)